### ISSUE TITLE
Accessibility. Game is not accessible for blind players

### ISSUE THREAD

**Issue by sevapopov2** [Grimmys/rpg_tactical_fantasy_game#69]
Hello,
First, thank you for this project, it can be a good base and reference for me to use in case of learning about audio game development.
But there is an issue:
I am a blind player and this game is not accessible for me as for screen reader user.
You can use the following library to provide accessibility for screen readers and keyboard access:
[accessible_output2](https://github.com/accessibleapps/accessible_output2)
Thank you!

**Comment by Grimmys** [comment]
Hello @sevapopov2,

That's true that accessibility was never really taken into account for this project, and that's a shame.
To be honest, I never really implemented anything like this, so it would be new for me, but I'm really looking forward to learn how to do it.

Thanks @JimZhouZZY btw for your implication.

>  The screen reader requires us converting all displayed characters to text output (and then to audio output), which is a lot of work and might take some time to done.

Hmmm I guess there should be a way to automate this process.
Especially knowing that soon every string displayed on the screen should be located around the same place (because of internalization system), I guess all the strings from there could be sent to the screen reader?

But yeah still it will require time to investigate how to do it properly.

**Comment by sevapopov2** [comment]
Hello,
Thank you for your answers!
You can ask on [audiogames forum](https://forum.audiogames.net) for consultation about providing accessibility of the game. I am sure people will help with that!
From my point I am ready to test the project and report bugs when accessibility starts implementing!
Thank you!

**Comment by andytorrestb** [comment]
These features might be useful to make this possible (to be explored after stable-release):
1. Discover mode to read what is on the map (number/type of enemies, obstacles, chests, buildings, etc). 
2. W/A/S/D can be used to control a logical cursor and read information to users (non-visual map exploration).
3. Hotkeys for ally and enemy units. Allow users to report proximity of items using radial distance (n units).
4. Allow users to confirm/undo a move (definitely useful outside of blind mode).
5. establish TTS verbosity (n units, vs unit types, vs unit stats). Alternatively we can help them export unit stats to braille?
6. Summarize the battlefield at the start of each phase within a turn.
7. Inform users of potential critical hits or type advantages. 
8. Try to curate distinct sounds for classes actions etc.
